sample/src/main/java/com/decathlon/design/sample/components/screens/DialogsFragment.kt

```diff
@@ -0,0 +1,120 @@
+package com.decathlon.design.sample.components.screens
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.decathlon.design.sample.R
+import com.decathlon.design.sample.databinding.FragmentDialogsBinding
+import com.decathlon.vitamin.modals.VitaminAlertDialogBuilder
+import com.decathlon.vitamin.modals.VitaminAlertDialogBuilderCentered
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+class DialogsFragment : ComponentFragment() {
+
+    private lateinit var binding: FragmentDialogsBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentDialogsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.ad.setOnClickListener {
+            this.context?.let {
+                VitaminAlertDialogBuilder(it)
+                    .setTitle("Subtitle 1")
+                    .setMessage("Body 3")
```

Comment:
Maybe add a small `lorem ipsum ` in all the dialogs messages to show the multiline?